machines/orchid-inca/eluthia-cron-example/build.py

```diff
@@ -0,0 +1,34 @@
+from eluthia.decorators import chmod, copy_folder, file, git_clone, empty_folder
+from eluthia.defaults import control
+from eluthia.functional import pipe
+from eluthia.py_configs import deb822
+
+
+@chmod(0o644)
+@file
+def cron_file(full_path, package_name, apps):
+    return f'''\
+        0 * * * * EMAIL_USER={apps[package_name]['env']['EMAIL_USER']} EMAIL_PASSWORD={apps[package_name]['env']['EMAIL_PASSWORD']} root python3 /usr/local/bin/eluthia-cron-example/main.py
```

Comment:
root python3? I don't think that root there will work. No such command called root.